08-verification-right.md

```diff
@@ -110,29 +111,27 @@ Run the code by running this command in your terminal, still within your `kilt-r
 node claim-with-signed-nonce.js
 ```
 
-You should see in your logs the `dataToVerify`.
+You should see in your logs the `dataToVerifyJSONString`, which is a string representation of the data to verify.
 
 Copy it, you'll need it in the next step.
 
 ## As the <span class="label-role verifier">verifier</span>: verify the `signedNonce` and `attestedClaim`
 
 Create a new file `verification-with-nonce.js`.
 
-Paste the following code into it (make sure to replace `<dataToVerifyJSONString>` and `<nonce>` with the relevant objects):
+Paste the following code into it (make sure to replace `<dataToVerifyJSONString>` and `<nonce>` with the values obtained in the previous steps):
 
 <!-- copy and paste 2️⃣ verifyNonce_example from 6_verification-with-nonce.ts -->
 
 <!-- IMPORTANT! Respect the UNCOMMENT-LINE and REMOVE-LINE comments -->
 
 ```javascript
-const Kilt = require('@kiltprotocol/sdk-js')
+const KiltUtils = require('@kiltprotocol/utils')
```

Comment:
I'd advocate importing everything from '@kiltprotocol/sdk-js' (which just reexports from the other packages) to avoid confusing the new user with multiple packages.

Reply:
It does not export the utils package tho...

Reply:
That is why I thought we might need to import the utils package for the crypto operations.